appserver/routes.go

```diff
@@ -13,6 +13,9 @@ func (srv *AppServer) SetupRoutes() error {
 			Methods("GET")
 
 		// ADD YOUR OWN ROUTES HERE
+		srv.router.HandleFunc("/topology", srv.HandleTopology()).
```

Comment:
Can the namespace be taken here as a query parameter?

Reply:
Something like this `Queries("namespace", "{[a-z]*?}").`

Reply:
Thanks Avni, i didn't put it in the description, but i haven't finished this part yet, it's on the TODO list to figure out how we can pass the namespace, url and bearerToken. I will definitely pass the namespace as a query parameter as you mentioned in our earlier call. Stayed tuned! :)

Reply:
updated here: https://github.com/redhat-developer/app-service/pull/3#discussion_r292866552